rust/cx-sdk/build.rs

```diff
@@ -0,0 +1,20 @@
+//! Build script for the cx-sdk crate.
+use std::process::{Command, Stdio};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let rustc_version_command = Command::new("rustc")
```

Comment:
Use the env variable instead of hardcoding rustc

Reply:
https://doc.rust-lang.org/cargo/reference/environment-variables.html

> RUSTC_WRAPPER — Instead of simply running rustc, Cargo will execute this specified wrapper, passing as its command-line arguments the rustc invocation, with the first argument being the path to the actual rustc. Useful to set up a build cache tool such as sccache. See build.rustc-wrapper to set via config. Setting this to the empty string overwrites the config and resets cargo to not use a wrapper.